### ISSUE TITLE
[compdat] Add support for * notation in WELOPEN

### ISSUE THREAD

**Issue by lindjoha** [equinor/ecl2df#314]
I think the compdat export crashes when WELOPEN keyword contains * notation, f.ex like this:

```
WELOPEN
   'D_*'  'OPEN'  /
/
```

The Error message is also a bit confusing:

```
  File "/prog/res/komodo/2021.05.01-py36-rhel7/root/lib/python3.6/site-packages/ecl2df/compdat.py", line 399, in applywelopen
    "(I=%s,J=%s,K=%s)" % (row["I"], row["J"], row["K"])
ValueError: A WELOPEN keyword is not acting on any existing connection. (I=None,J=None,K=None)
```

**Comment by berland** [comment]
Duplicate of #218 ?

**Comment by lindjoha** [comment]
It is certainly related, but I don't know if it's exactly the same. I was not using WLIST when I got this error

**Comment by berland** [comment]
No, it is not exactly the same.

**Comment by lindjoha** [comment]
Solved in PR #333
